[INFO -- 2017-02-26 14:07:30]
Starting flip_value script
version: 2.0
[INFO -- 2017-02-26 14:07:30]
Initialization strings:
[INFO -- 2017-02-26 14:07:31]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-02-26 14:07:31]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-26 14:07:31]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-26 14:07:31]
(4) file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-26 14:07:31]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-26 14:07:31]
(6) set args -s 2048 -n 228 -i /micNfs/gdb-fault-injection/injector/bin/lud/input_2048_th_228 -g /micNfs/gdb-fault-injection/injector/bin/lud/gold_2048_th_228
[INFO -- 2017-02-26 14:07:31]
initSignal:2
[INFO -- 2017-02-26 14:07:31]
endSignal:4
[INFO -- 2017-02-26 14:07:31]
seqSignal:30
[INFO -- 2017-02-26 14:07:31]
faultMode:0
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 14:07:41]
pythonException: invalid frame
[INFO -- 2017-02-26 14:07:41]
Backtrace BEGIN:
#0  0x0000000000404ccd in L_lud_omp_123__par_loop2_2_5 () at lud_omp.c:148
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffd9c0 in ?? ()
#3  0x00007ffff4ea0100 in ?? ()
#4  0x0000000000000001 in ?? ()
#5  0x0000000000000003 in ?? ()
#6  0x0000000000000000 in ?? ()

143	            for (i = 0; i < BS; i++)
144	            {
145	                for (k=0; k < BS; k++) {
146	#pragma omp simd 
147	                    for (j = 0; j < BS; j++) {
148	                        sum[j] += temp_left[BS*i + k] * temp_top[BS*k + j];
149	                    }
150	                }
151	#pragma omp simd 
152	                for (j = 0; j < BS; j++) {

Backtrace END
Fliping array at pos: 14
Memory content before bitflip:0xda0xed0xc10x3b
Fault Mode: Single bit-flip
base address to flip value: 0x7fffffffc878
address max offset: 4
Memory content after  bitflip:0xda0xed0xc10x7b
frame name: L_lud_omp_123__par_loop2_2_5
symbol name: sum
symbol filename: lud_omp.c
symbol line: 130
value: {0.00750970189, 0.00750212604, 0.00749453064, 0.00748727378, 0.00747945672, 0.00747214025, 0.00746475952, 0.00745725026, 0.00595402857, 0.00594788743, 0.00594207086, 0.00593593484, 0.00593013922, 0.00592424674, 2.01387499e+36, 0.00591244362}
value address: 0x7fffffffc840
Type: The type is an array.
Type sizeof: 64
Field name: None
Field Type: A range type, that is, an integer type with bounds.
Field Type sizeof: 1
Field Type range: (0L, 15L)

[INFO -- 2017-02-26 14:07:41]
Fault Injection Successful after 10.4094009399s
[INFO -- 2017-02-26 14:07:53]
event type: exit
[INFO -- 2017-02-26 14:07:53]
exit code: 1
